frontend/src/modules/conversation/components/conversation-list-table.vue

```diff
@@ -33,9 +33,10 @@
               }"
               class="flex items-center text-black"
             >
-              <span class="font-semibold truncate">{{
-                scope.row.title
-              }}</span>
+              <span
+                class="font-semibold truncate"
+                v-html="scope.row.title"
```

Comment:
I don't think titles should contain (and render) HTML

We don't gain that much apart from possible weird UI inconsistencies, and I think that we should protect our UI/UX and the overall customer experience and kinda enforce the user to have plain text-based titles.

Otherwise once we add for example a Search bar, displaying titles that might contain HTML would be a painful thing.

Reply:
The problem is that when conversation is created on the backend we use `activity.crowdInfo.body` as title and with Dev.to this is an HTML/rich text comment.

Reply:
That's why I think it should be text only 😬 so that it looks ugly and we "gently force" the user to set a proper title to it.

Reply:
Maybe we should expand this conversation to Slack, I feel it could be an important one

Reply:
Will add three-line truncate like discussed on slack.